app/src/main/res/xml-v31/accessibility_service_config.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<accessibility-service xmlns:android="http://schemas.android.com/apk/res/android"
+    android:accessibilityEventTypes="typeAllMask"
+    android:accessibilityFeedbackType="feedbackSpoken|feedbackAudible|feedbackHaptic"
+    android:accessibilityFlags="flagDefault|flagRequestTouchExplorationMode|flagRetrieveInteractiveWindows|flagReportViewIds|flagEnableAccessibilityVolume"
+    android:canRequestTouchExplorationMode="true"
+    android:canRetrieveWindowContent="true"
+    android:isAccessibilityTool="true"
```

Comment:
Thank you very much! I wasn't aware of the need to declare this when targeting for api 31, it would be bullshit when uploading to the Play Store.

Reply:
From what I researched, it is possible to specify this in the manifest as well.

https://stackoverflow.com/questions/70312896/declaring-app-as-not-an-accessibilitytool-to-comply-with-google-play-requirement